storage/cost-analysis/storage_soft_delete_relative_cost_analyzer.py

```diff
@@ -21,10 +21,10 @@
 storage/v2/total_byte_seconds
 metric.
 
-Relative cost of each bucket = ('soft delete retention duration'
-                                 × 'deleted bytes' / 'total bytes seconds' )
-                                 x 'cost of storing in storage class'
-                                 x 'ratio of storage class'.
+Relative cost of each bucket = deleted_bytes / total_byte_seconds
```

Comment:
I think parentheses here do clarify the math - in my experience some people have different intuition on how to process division and multiplication order of operations.

Reply:
I agree but in this situation, the order of operations (multiplication 'X' and division '/') doesn't matter because only those two operations are involved. The final result will be the same regardless of which operation is performed first.